notification-manager/src/main/resources/application.yaml

```diff
@@ -0,0 +1,6 @@
+spring:
+  application:
+    name: notification-manager
+
+server:
+  port: 8083
```

Comment:
this is for DEV. Check other services, if this config in default one, move it to DEV as well